mail/notmuch/files/notmuch-0.29-configure.patch

```diff
@@ -5,7 +5,7 @@
  }
  EOF
 -    if ! TEMP_GPG=$(mktemp -d); then
-+    if ! TEMP_GPG=$(mktemp -d -t /tmp); then
++    if ! TEMP_GPG=$(mktemp -d ./XXXXX); then
```

Comment:
Why not use something like `mktemp -d /tmp/notmuch.XXXXXX` to keep this socket in `/tmp`?

Reply:
Thanks. I expected`/usr/bin/mktemp -d /tmp/notmuch.XXXXXX` to have the same result as the `-t /tmp` variant, but on macOS 10.14.5 it actually returns a different result. I have pushed that change, now to see what buildbot makes of it.

Reply:
The azure builds are at least OK with this. Note these only test macOS 10.13 and 10.14.
The travis builds are ongoing. These are more likely to fail (or timeout) for unrelated reasons but lets at least give them change to run. In the end though I think nothing here will test macOS10.10 or older, so to check there the simplest is to just merge this and see what happens.

Reply:
> I think nothing here will test macOS10.10 or older, so to check there the simplest is to just merge this and see what happens.

That sound reasonable.

Reply:
Actually, looking back at past PRs, I believe travis will must eventually timeout here, so there is I think little to learn now from waiting for it. I am going to merge, and then lets see what the buildbots say.